src/index.html

```diff
@@ -1,12 +1,42 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Kaushan+Script&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+  <link rel="shortcut icon" type="image/jpg" href="images/favicon.png">
+  <script src="https://kit.fontawesome.com/b433974680.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet"  href="style.css" />
+
+  <title>
+    🚲CicloVida
```

Comment:
```suggestion
    CicloVida
```
esto pudiese generar errores